src/Modal/Modal.stories.tsx

```diff
@@ -11,8 +12,9 @@ storiesOf('molecules|Modal', module)
       <Modal
         title='Disclaimer'
         buttons={[
-          <Button onClick={() => { return }}>Whatsup</Button>,
-          <Button danger onClick={() => { return }}>Delete</Button>
-        ]}>{loremIpsum}</Modal>
+          <Button onClick={() => { return }} key={1}>Whatsup</Button>,
```

Comment:
Can you refactor this so that Modal maps and clones the button to inject them with key attribute.

```React.Children.map(buttons, (child, index) => React.cloneElement(child, { ...child.props, key: index })) ```

Reply:
nit: we can now get rid of the `key={}` here